### ISSUE TITLE
Token for conference room 

### ISSUE THREAD

**Issue by ashrafbendary** [ant-media/Ant-Media-Server#3797]
**Is your feature request related to a problem? Please describe.**
the normal 3 token typs work with streamid not roomid so when you activate any type of it its automatically stop conference
  
**Describe the solution you'd like**
token work with roomid not streamid

**Comment by mekya** [comment]
Yes, this feature is available through multi-track conferencing.  There will be a blog post published about multi-track conferencing.
